ios/PresentationLayer/UIToolkit/Sources/UIToolkit/BaseViews/SwiftUI/Button/SlidingButton/SlidingButton.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Created by David Sobíšek on 23.11.2023
+//  Copyright © 2023 Matee. All rights reserved.
+//
+
+import SwiftUI
+
+struct SlidingButton: View {
+    
+    private let title: String
+    private let buttonIcon: Image
+    private let color: Color
+    private let isLoading: Bool
+    private let maxWidth: CGFloat
+    private let action: () -> Void
+    
+    init(
+        title: String,
+        buttonIcon: Image,
+        color: Color,
+        isLoading: Bool,
+        maxWidth: CGFloat,
+        action: @escaping () -> Void = {}
+    ) {
+        self.title = title
+        self.buttonIcon = buttonIcon
+        self.color = color
+        self.isLoading = isLoading
+        self.maxWidth = maxWidth
+        self.action = action
+    }
+    
+    var body: some View {
+        ZStack(alignment: .leading) {
+            RoundedRectangle(cornerRadius: 25)
+                .fill(color)
+                .frame(height: 50)
+            
+            Text(title)
```

Comment:
I would prefer text aligned on the center of slider :D

Reply:
In this example I prefer it where it is rn, you can put it in the middle when you use it elsewhere😀